composer/src/composer.rs

```diff
@@ -1573,7 +1622,7 @@ impl ComposeTest {
                 ),
             })?;
         if !self.reuse || self.prune_matching {
-            tracing::debug!("Starting container: {}, ip: {}", name, id.1);
+            tracing::debug!("Starting container: {}, ip: {:?}", name, id.1);
```

Comment:
This can be reverted?

```suggestion
            tracing::debug!("Starting container: {}, ip: {}", name, id.1);
```